.github/actions/install-intel-dcap/action.yml

```
@@ -0,0 +1,15 @@
+name: 'Install Intel DCAP libraries'
+inputs:
+  ubuntu-version:
+    required: true
+runs:
+  using: "composite"
+  steps:
+  - name: Install TDX dependencies
+    id: install-tdx-dependencies
+    shell: bash
+    run: |
+      sudo curl -sL https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo gpg --dearmor --output /usr/share/keyrings/intel-sgx.gpg
+      sudo echo 'deb [arch=amd64 signed-by=/usr/share/keyrings/intel-sgx.gpg] https://download.01.org/intel-sgx/sgx_repo/ubuntu ${{ inputs.ubuntu-version }} main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
```

thanks a lot for this! (I think it's better like this)

```suggestion
      curl -sL https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo gpg --dearmor --output /usr/share/keyrings/intel-sgx.gpg
      echo 'deb [arch=amd64 signed-by=/usr/share/keyrings/intel-sgx.gpg] https://download.01.org/intel-sgx/sgx_repo/ubuntu ${{ inputs.ubuntu-version }} main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
```

---

@mkulke I think I saw you accepting this suggestion but it's now missing?